.github/release.yml

```
@@ -0,0 +1,32 @@
+name: Upload a new release
+on:
+  push:
+    tags:
+      - "v*" #Napptive uses Semantic Versioning with the prefix 'v'
+jobs:
+  docker:
+    name: Push Docker Release Images
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Get the version
+        id: get_version
+        run: echo "version=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_OUTPUT
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKER_HUB_USER }}
+          password: ${{ secrets.DOCKER_HUB_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          push: true
+          tags: cdelope/test-action:${{ steps.get_version.outputs.version }}
```

I think naming is incorrect

```suggestion
          tags: cdelope/test-action:${{ steps.get_version.outputs.version }}
```

---

sure! changed to `napptive/playground-action`